src/components/banner/banner.config.yml

```diff
@@ -36,7 +36,7 @@ context:
 variants:
   - name: default
     label: Large, content centered
-    hidden: false
+    hidden: true
```

Comment:
Do we need to set this on variants? If so, does it need to be set on the other variants?

Reply:
Good catch! Thanks!